src/UnitySampleProject/Assets/Samples/web3.unity SDK/2.5.0/Web3.Unity Samples/Scripts/Scenes/SampleMain/Marketplace/MarketplaceCalls.cs

```diff
@@ -0,0 +1,134 @@
+using ChainSafe.Gaming.UnityPackage;
+using ChainSafe.Gaming.UnityPackage.Model;
+using Newtonsoft.Json;
+using Scripts.EVM.Remote;
+using UnityEngine;
+
+/// <summary>
+/// Marketplace sample calls for use with the api documentation.
+/// Marketplace Api: https://docs.gaming.chainsafe.io/marketplace-api/docs/marketplaceapi
+/// Token Api: https://docs.gaming.chainsafe.io/token-api/docs/tokenapi
+/// </summary>
+public class MarketplaceCalls : MonoBehaviour
+{
+    #region fields
+
+    #region Collections
+    
+    private string collectionId721 = "d588268b-8a5b-486a-8ea1-4122b510d71e";
+    private string collectionId1155 = "ebeaaee5-f7c2-4561-abb9-60ba749db7cd";
+    private string marketplaceId = "4986983b-2bcc-4bb3-b0db-a3448fbdee2b";
+    private string tokenId = "0";
+
+    #endregion
+    
+    #endregion
+
+    #region Methods
+    
+    #region MarketplaceCalls
+    
+    /// <summary>
+    /// Gets all items in a project.
+    /// Path: https://api.gaming.chainsafe.io/v1/projects/{projectID}/items
+    /// </summary>
+    public async void GetProjectItems()
+    {
+        string path = $"/items?chainId={Web3Accessor.Web3.ChainConfig.ChainId}";
+        var jsonResponse = await CSServer.GetData(path);
+        MarketplaceModel.Root response = JsonConvert.DeserializeObject<MarketplaceModel.Root>(jsonResponse);
+        CSServer.PrintObject(response);
```

Comment:
The return value can be the response `public async MarketplaceModel.Root GetProjectItems`

Reply:
Same for all other methods in this class

Reply:
done